etc/WMAgentConfig.py

```diff
@@ -202,6 +203,7 @@
 config.JobSubmitter.skipRefreshCount = 20  # (If above the threshold meet, cache will updates every 20 polling cycle) 120 * 20 = 40 minutes
 config.JobSubmitter.submitScript = os.path.join(os.environ["WMCORE_ROOT"], "etc/submit.sh")
 config.JobSubmitter.extraMemoryPerCore = 500  # in MB
+config.JobSubmitter.condorJobsFraction = 0.75  # fraction of the condor schedd limit
```

Comment:
Remove it since it's already in the aux db

Reply:
Is this already in aux db?

Reply:
Never mind, I was reviewing commit by commit and didn't see the other commit. :-)